src/common/models/ContractEntity.ts

```diff
@@ -38,6 +40,12 @@ export enum EntityType {
   RedeemIntentConfirmeds = 'redeemIntentConfirmeds',
   RedeemProgresseds = 'redeemProgresseds',
   UnstakeProgresseds = 'unstakeProgresseds',
+  // M1 entities
+  DeclaredDepositIntents = 'declaredDepositIntents',
+  ConfirmedWithdrawIntents = 'confirmedWithdrawIntents',
+  DeclaredWithdrawIntents = 'declaredWithdrawIntents',
+  ConfirmedDepositIntents = 'confirmedDepositIntents',
+  CreatedUtilityTokens = 'createdUtilityTokens',
```

Comment:
Pls add `AvailableStateRoots` and `ProvenGateways` also

Reply:
Added